<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetRelatorio</class>
 <widget class="QWidget" name="WidgetRelatorio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Vendedores</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2" rowspan="4" colspan="3">
       <widget class="TableView" name="tableRelatorio">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBoxTotais">
     <property name="title">
      <string>Totais</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTotalVendedor">
        <property name="text">
         <string>Vendedores</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTotalLoja">
        <property name="text">
         <string>Loja</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="TableView" name="tableTotalLoja">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="TableView" name="tableTotalVendedor">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxValorComissao">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string>Comissão    R$ </string>
          </property>
          <property name="maximum">
           <double>99999999.989999994635582</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxPorcentagemComissao">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGeral">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string>Geral    R$ </string>
          </property>
          <property name="maximum">
           <double>99999999.989999994635582</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBoxResumoOrcamento">
     <property name="title">
      <string>Resumo Orçamentos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="TableView" name="tableResumoOrcamento">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filtros</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QDateEdit" name="dateEditMes">
        <property name="displayFormat">
         <string>MM/yy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mês:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonExcel">
        <property name="text">
         <string>Gerar Excel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
